File: P1/Ejercicio1/BicicletaCarretera.java
package Ejercicio1;

public class BicicletaCarretera extends Bicicleta{
    private static int id= 0;
    
    public BicicletaCarretera() {
        super(id);
        ++id;
    }

    @Override
    public String toString() {
        return ("Bicicleta de carretera con identificador de carrera " + getIdCarrera());
    }
}

File: P1/Ejercicio2/BicicletaCarretera.py
from Bicicleta import Bicicleta

class BicicletaCarretera(Bicicleta):
    id_carrera = 0

    def __init__(self):
        super().__init__()

    def __str__(self):
        return f"Bicicleta de carretera con identificador de carrera {self.get_id_carrera()}"
    
